src/components/DataTable/index.jsx

```diff
@@ -7,13 +7,15 @@ import {
   oneOf,
   oneOfType,
   object,
+  bool,
```

Comment:
Why are we adding pagination? Is the last fire endpoint paginated (i.e., uses continuation token)?

Reply:
As per discussion with Dustin in irc we are keeping as many as 100 lastfire info rows for each hookId (expiration logic). Hence I think showing all rows at once won't be good.

Reply:
We also only purge those once per day, so a particularly active hook might have many more rows than that at some time during the day!